crypto/x509/x_crl.c

```diff
@@ -103,6 +103,17 @@ static int crl_set_issuers(X509_CRL *crl)
         }
 
         if (gtmp != NULL) {
+            /*
+             * Validation to ensure Certificate Issuer extensions in CRL
+             * entries only appear when the Indirect CRL flag is TRUE in the
+             * Issuing Distribution Point (IDP) extension, as required by
+             * RFC 5280 section 5.3.3.
+             */
+            if (!crl->idp || !crl->idp->indirectCRL) {
```

Comment:
Style nit: For pointer types we always explicitly test against NULL, i.e. `crl->idp == NULL`